branch-validation.yml

```diff
@@ -171,13 +171,14 @@
       set_fact: master=true
       when: floating_ip in ansible_all_ipv4_addresses
 
-    # This is manual for now, not keen on automated kernel panic
+    # initiate hard reboot on the current master controller
     - name: initiate a failover on controller 0
-      pause: prompt="On ucarp {{ inventory_hostname }} $ echo c > /proc/sysrq-trigger (press enter when done)"
+      shell: echo 'b' | tee /proc/sysrq-trigger
       when: master is defined
 
     - name: initiate a failover on controller 1
-      pause: prompt="On ucarp {{ groups['controller'][1] }} $ echo c > /proc/sysrq-trigger (press enter when done)"
+      delegate_to: "{{ groups['controller'][1] }}"
```

Comment:
Minor nit. The delegate_to should be the last key on the task hash, just a style thing.